std_msgs/Header header
sensor_msgs/Range[] array
